scripts/cba/prepare_toot_reference.py

```diff
@@ -0,0 +1,260 @@
+
+For TOOT methodology, the reference network should contain ALL CBA projects.
+The simple_2030.nc network already contains projects where in_reference2030=True.
+This script adds the missing projects (where in_reference2030=False) to create
+the complete TOOT reference network with all projects included.
+
+Handles reference year to planning horizon mapping:
+- reference2030 → horizon 2030
+- reference2035 → horizon 2040 (if 2035 not available)
+"""
+
+import logging
+
+import pandas as pd
+import pypsa
+
+from scripts._helpers import configure_logging, set_scenario_config
+from scripts.cba.project_helpers import (
+    get_project_status_for_horizon,
+    log_horizon_mapping,
+)
+
+logger = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    if "snakemake" not in globals():
+        from scripts._helpers import mock_snakemake
+
+        snakemake = mock_snakemake("prepare_toot_reference")
+
+    configure_logging(snakemake)
+    set_scenario_config(snakemake)
+
+    # Load simplified network
+    n = pypsa.Network(snakemake.input.network)
+
+    # Load project definitions
+    transmission_projects = pd.read_csv(snakemake.input.transmission_projects)
+
+    # Get planning horizons from config
+    planning_horizons = snakemake.config["scenario"]["planning_horizons"]
+    current_horizon = int(snakemake.wildcards.planning_horizons)
+
+    logger.info(f"\n{'=' * 80}")
+    logger.info("PREPARING TOOT REFERENCE NETWORK")
+    logger.info(f"{'=' * 80}")
+    logger.info(f"Current planning horizon: {current_horizon}")
+    logger.info(f"Available planning horizons: {planning_horizons}")
+
+    # Log reference mapping
+    log_horizon_mapping(planning_horizons)
+
+    # For TOOT: Add projects that are NOT in the reference (in_reference=False)
+    # the reference network should have ALL projects
+    projects_added = []
+    projects_created = []
+    projects_in_base = []
+    projects_missing_buses = []
+
+    # Add transmission projects for this horizon
+    for _, project in transmission_projects.iterrows():
+        in_reference = get_project_status_for_horizon(
+            project, current_horizon, planning_horizons
+        )
+
+        bus0 = project["bus0"]
+        bus1 = project["bus1"]
+        link_id = f"{bus0}-{bus1}-DC"
+        reverse_link_id = f"{bus1}-{bus0}-DC"
+
+        # Check if both buses exist in the network
+        buses_exist = bus0 in n.buses.index and bus1 in n.buses.index
+
+        if not buses_exist:
+            # Buses don't exist - skip project (e.g., extra-EU connections)
+            projects_missing_buses.append(project["project_id"])
+            logger.warning(
+                f" Project {project['project_id']} ({project['project_name']}): Buses not in network"
+            )
+            logger.warning(
+                f"    Missing buses: {bus0 if bus0 not in n.buses.index else ''} {bus1 if bus1 not in n.buses.index else ''}"
+            )
+            continue
```

Comment:
Not necessary any longer, since i added: 82c3dd087eabea70ca0c04a527758ac8afef34ed